docs/pages/enroll-resources/kubernetes-access/kubernetes-access-examples.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Examples of Role-Based Access Control in Kubernetes
+description: RBAC examples with Kubernetes.
+---
+
+In Teleport's flexible RBAC system, you can define custom roles and granularly control what users can do within your Kubernetes clusters. 
+Limit access to specific namespaces, enforce read-only permissions, or restrict administrative actions as needed.
+
+:Note:
+Please reference [Setting Up Teleport Access Controls for Kubernetes](../../enroll-resources/kubernetes-access/manage-access.mdx) before proceeding, if you have not done so already.
```

Comment:
```suggestion
<Notice type="tip">
Reference [Setting Up Teleport Access Controls for Kubernetes](../../enroll-resources/kubernetes-access/manage-access.mdx) before proceeding, if you have not done so already.
</Notice>
```
- Using our docs engine syntax to create a notice
- Removing "Please", which I don't think we need in technical writing